crates/astria-composer/tests/blackbox/composer.rs

```diff
@@ -5,7 +5,9 @@ use proto::generated::sequencer::v1alpha1::NonceResponse;
 use sequencer_types::AbciCode;
 use tendermint_rpc::{
     endpoint::broadcast::tx_sync,
-    request::{self,},
+    request::{
```

Comment:
```suggestion
    request,
```
I think this should work? Importing `request::self` was probably an oversight of a refactor.